pkg/operator/controller/names.go

```diff
@@ -19,6 +19,10 @@ const (
 	// the certificates for secure DNS metrics.
 	MetricsServingCertAnnotation = "service.beta.openshift.io/serving-cert-secret-name"
 
+	// TopologyAwareHintsAnnotationKey is the annotation key to enable topology aware hints
```

Comment:
May be include a link to https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/ or https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/2433-topology-aware-hints/README.md